src/bufr/BufrParser/Query/ResultSet.cpp

```diff
@@ -296,7 +283,7 @@ namespace bufr {
 
         // Convert dims per data frame to dims for all the collected data.
         dims[0] = totalRows;
-        if (dataFrames_.size() > 1)
+        if (dataFrames_.size() > 0)
```

Comment:
Why the change to > 0 instead of > 1?

Reply:
I was accidentally not slicing if there is only 1 dataframe in the file.

Reply:
By slicing I mean removing unwanted dimensional data from the dimensions array.